docs/robots/husky/index_husky.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Husky
+sidebar_position: 1
+---
+
+import ComponentIntroductionHusky from "/components/introduction_husky.mdx";
+
+<ComponentIntroductionHusky />
+
+TODO
```

Comment:
Is the "TODO" here intentional?

Reply:
Yes, this TODO is intentional. 
I think updating the Husky section requires a dedicated branch, for tutorials and splitting up the User Manual into separate pages (Maintenance, Integration, etc)